src/py/flwr/client/mod/__init__.py

```diff
@@ -15,12 +15,8 @@
 """Mods."""
 
 
-from .centraldp_mods import fixedclipping_mod
+from .centraldp_mods import adaptiveclipping_mod, fixedclipping_mod
 from .secure_aggregation.secaggplus_mod import secaggplus_mod
 from .utils import make_ffn
 
-__all__ = [
-    "make_ffn",
-    "secaggplus_mod",
-    "fixedclipping_mod",
-]
+__all__ = ["make_ffn", "secaggplus_mod", "fixedclipping_mod", "adaptiveclipping_mod"]
```

Comment:
```suggestion
__all__ = [
    "adaptiveclipping_mod",
    "fixedclipping_mod",
    "make_ffn",
    "secaggplus_mod",
]
```
I would recommend adding a comma to the end of each line to make `__all__` in a multiple-line format for consistency. Plus, the lines should be sorted in dictionary order.